README.md

```diff
@@ -43,8 +45,7 @@ Start the application with:
 npm run startup [--paas]
 ```
 
-Note : use the ```--paas``` flag to run against the verify-service-provider-dev instance on paas.
-Otherwise it will use the value of the VERIFY_SERVICE_PROVIDER_HOST environment variable, defaulting to `http://localhost:50400`.
+Note : use the ```--paas``` flag to run against the verify-service-provider-dev instance on GOV.UK Platform as a Service (PaaS). Otherwise the application will use the value of the VERIFY_SERVICE_PROVIDER_HOST environment variable, defaulting to `http://localhost:50400`.
```

Comment:
We probably don't want people to run against our PaaS instance - we should either not document this bit (!!!) or we should decide what we want people cloning this to do. It might we worth moving this stuff into local-startup for Verify developers or something. Let's talk about this as the whole building stuff for open source is really difficult/interesting

Reply:
Spoke to @richardTowers about this. He's happy to keep here for the mo until we have reason to remove.

Reply:
I've removed the reference to pass, and explained what defaults are used instead. There is still reference to the paas flag in the package.json scripts to remind us about it